nb-configuration.xml

```diff
@@ -1,32 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project-shared-configuration>
-	<!--
```

Comment:
I don't think anything has changed in this file, except the formatting, so please revert this file to the initial version

Reply:
Reverted.

Reply:
Sorry, but it does not seem to be reverted, but it's fine, I'll do it after the merge 👍